### PR TITLE
Added a new Python question

### PR DESCRIPTION
**Pull request by NeelPatel31**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/full-quiz.js [hunk 1]
```diff
@@ -1,4 +1,13 @@
 const fullQuiz = [
+  {
+    Question: "What is the data type of (1) and (1,) ?",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:3]: Proposed fix: What are the outputs for type(1) and type((1,)) in Python?

File: src/data/full-quiz.js [hunk 2]
```diff
@@ -1,4 +1,13 @@
 const fullQuiz = [
+  {
+    Question: "What is the data type of (1) and (1,) ?",
+    Answer: "<class 'int'>  <class 'tuple'>",
+    Distractor1: "<class 'tuple'> <class 'int'>",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:5]: I think we should change this to include a more obvious wrong answer like a data type of string. 
Proposed fix: 
 Distractor1: "<class 'str'> <class 'int'>",

File: src/data/full-quiz.js [hunk 3]
```diff
@@ -1,4 +1,13 @@
 const fullQuiz = [
+  {
+    Question: "What is the data type of (1) and (1,) ?",
+    Answer: "<class 'int'>  <class 'tuple'>",
+    Distractor1: "<class 'tuple'> <class 'int'>",
+    Distractor2: "<class 'int'>  <class 'tuple'>",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:6]: The distractors should not be the same as the correct answer.
The distractors will always be the wrong answer. 

Proposed fix: 
Distractor2: "<class 'dictionary'>  <class 'list'>",

File: src/data/full-quiz.js [hunk 4]
```diff
@@ -1,4 +1,13 @@
 const fullQuiz = [
+  {
+    Question: "What is the data type of (1) and (1,) ?",
+    Answer: "<class 'int'>  <class 'tuple'>",
+    Distractor1: "<class 'tuple'> <class 'int'>",
+    Distractor2: "<class 'int'>  <class 'tuple'>",
+    Distractor3: "<class 'int'> <class 'int'> ",
+    Explanation: "Here (1) means only 1 but if we add coma(,) then it will becomes a tuple.",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:8]: type(1) will return <class 'int'> and type((1,)) will return <class 'tuple'>

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Hi @NeelPatel31 ! 

Thank you for your Python question.
Please look at the requested changes. 👍

**Review by NeelPatel31 — COMMENTED** [review]
Changed as per the requested.

**Review by jdwilkin4 — APPROVED** [review]
LGTM! 👍

**Comment by jdwilkin4** [comment]
Hi @NeelPatel31 ! 

Congrats on your first PR for the developer quiz site.
We look forward to future contributions. 👍

**Pull request merged**